ci/run-on-linux.sh

```diff
@@ -3,7 +3,7 @@
 # If we try to use a normal Github Actions container with
 # github-pages-deploy-action, it will fail due to inability to find git.
 
-docker run -h test.box \
+docker run -h test.krbtest.com \
```

Comment:
I imagine this broke something?

Reply:
Yea on Heimdal it seemed to always just default to the `BOX` realm and I couldn't figure out a way to override that behaviour. Just seemed to work but making sure the hostname ended with the same realm as what we are testing with.